rfpremier/ah-django
# #164857207 Setup Postgres database

**What does this PR do?**

Sets up Postgres database as the database engine for the project

**Tasks to be completed**

- [x] Set up Postgres database
- [x] Add script in settings.py to use Postgres database as the DB engine
- [x]  Update requirements.txt file

**How can this be manually tested**
1. Clone this repository and check out the branch ch/164857207-set-up-postgres-database

2. Create a new virtual environment and activate it. Also, install all packages in requirements.txt

3. Ensure you have Postgres database installed and running on port 5432

4. Create a new database and export the database URL to the environment using the variable name DATABASE_URL
example :  `export DATABASE_URL=postgresql://yourusername:yourpassword@localhost/dbname`

5. Run the following commands to update database
`python manage.py makemigrations` 
`python manage.py migrate`

6. Run the tests using
`python manage.py test`

7. To the run server use
`python manage.py runserver`

**Relevant Pivotal tracker stories** 
[#164857207](https://www.pivotaltracker.com/story/show/164857207)

**Screenshots**
N/A
